compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -243,16 +241,16 @@ impl<Prov> Allocation<Prov> {
     ///
     /// If `panic_on_fail` is true, this will never return `Err`.
     pub fn uninit<'tcx>(size: Size, align: Align, panic_on_fail: bool) -> InterpResult<'tcx, Self> {
-        let count128 = (size.bytes_usize() / 16) + 1;
+        let align_usize: usize = align.bytes().try_into().unwrap();
```

Comment:
This should be factored out into a helper method to share code with the other place you're trying to make an aligned boxed slice of a particular size.